### PR TITLE
PMM-9144-MySQL-inventory-page-for-AWS-RDS-or-Aurora: update add insta…

### PR DESCRIPTION
**Pull request by solovevayaroslavna**
https://github.com/percona-platform/grafana/pull/287
https://jira.percona.com/browse/PMM-914
PMM_Add_Instance
Updating image on add instance page

### CODE REVIEW

**Review by rasika-chivate — CHANGES_REQUESTED** [review]

File: docs/setting-up/client/aws.md [hunk 1]
```diff
@@ -203,7 +203,7 @@ For PostgreSQL, use the same method described above.
 
 1. In the PMM web interface, go to <i class="uil uil-cog"></i> *Configuration* → {{icon.inventory}} *PMM Inventory* → {{icon.addinstance}} *Add Instance*..
 
-2. Select *AWS RDS MySQL or Aurora MySQL -- Add a remote instance*.
+2. Select *Amazon RDS -- Add a remote instance*.
 
     !!! note alert alert-primary ""
```
[the remark this answers is not in the document]
Comment (rasika-chivate) [reply docs/setting-up/client/aws.md:208]: Thanks for the reply @rnovikovP. 

The following statement holds true if we are considering MYSQL only, but we are trying to generalise and not stick to just MYSQL. So how valid is the following statement:

"If you have Amazon RDS with a MySQL version prior to 5.5, REPLICATION CLIENT privilege is not available there and has to be excluded from the above statement."

[the remark this answers is not in the document]
Comment (rnovikovP) [reply docs/setting-up/client/aws.md:208]: I have no idea about specific 5.5 as it was EOLed a long time ago ...

[the remark this answers is not in the document]
Comment (rasika-chivate) [reply docs/setting-up/client/aws.md:208]: In that case I think would it be a good idea to remove the following statement: Please reaffirm @rnovikovP 

"If you have Amazon RDS with a MySQL version prior to 5.5, REPLICATION CLIENT privilege is not available there and has to be excluded from the above statement."

[the remark this answers is not in the document]
Comment (rnovikovP) [reply docs/setting-up/client/aws.md:208]: 🤦‍♂️ . line 146 yes, it needs to be removed  (can't do this in UI   for some reasons  ( )

also, please remove  146-151  lines, all as they  are "not accurate" and not related to the context

**Review by percona-csalguero — APPROVED** [review]

**Review by rnovikovP — APPROVED** [review]

**Review by rasika-chivate — APPROVED** [review]

**Comment by rnovikovP** [comment]
oops, this needs to go to the 2.26 branch as the `main` = "What we  have published on the website"  so  this  update  can be published any moment on the website, but the version this  change applicable not yet released

**Comment by percona-csalguero** [comment]
> oops, this needs to go to the 2.26 branch as the `main` = "What we have published on the website" so this update can be published any moment on the website, but the version this change applicable not yet released

Should I revert it until next version?

**Pull request merged**